Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/Networking/Forge/Networking/FacepunchNetworkingPlayer.cs

```diff
@@ -0,0 +1,17 @@
+﻿#if FACEPUNCH_STEAMWORKS
+using Steamworks;
+
+namespace BeardedManStudios.Forge.Networking
+{
+	public class FacepunchNetworkingPlayer : NetworkingPlayer
+	{
+		public UDPPacketManager PacketManager { get; private set; }
+
+		public FacepunchNetworkingPlayer(uint networkId, SteamId steamId, bool isHost, NetWorker networker) : base(networkId, "", isHost, null, networker)
```

Comment:
Shouldn't `SteamId` be `CSteamID`? Besides this the file looks identical to `SteamNetworkingPlayer` can't we use that instead of this file?